staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd_test.go

```diff
@@ -526,6 +526,44 @@ func TestScaleUpdateWithoutSpecReplicas(t *testing.T) {
 	}
 }
 
+func TestScaleUpdateWithoutResourceVersion(t *testing.T) {
```

Comment:
can you set up a second test (e.g. TestScaleUpdateWithoutResourceVersionWithConflicts) that does this in the presence of conflicting updates to make sure no conflict error is returned to the client and their scale update is successful?

to set up the conflicting updates, start two goroutines:
* one that continuously submits a patch that updates a label and verifies the label update was effective
* one that continuously submits a scale update without a resourceVersion for a monotonically increasing replica value and verifies the scale update was effective

Let them run for a few seconds or for 100 updates or something. Neither request should receive a conflict error back from the server.

To verify this is testing what you want it to, set `retryOnConflict` to false in the storage impl, and the scale update should start encountering conflict errors.